frontend/packages/dev-console/src/components/import/app/AppSection.tsx

```diff
@@ -10,7 +10,11 @@ export interface AppSectionProps {
 
 const AppSection: React.FC<AppSectionProps> = ({ project }) => {
   return (
-    <FormSection title="App" divider>
+    <FormSection
+      title="General"
+      subTitle="NEEDS UX [Some help text about the create project section.]"
```

Comment:
Remove this.

Reply:
This is based on the UX designs.

Reply:
I removed the subTitle. I left the change to the title.
I guess we can wait for a final on the subtitle and then add it.